pkg/minikube/machine/advice.go

```diff
@@ -39,7 +39,8 @@ func MaybeDisplayAdvice(err error, driver string) {
 	if errors.Is(err, oci.ErrExitedUnexpectedly) || errors.Is(err, oci.ErrDaemonInfo) {
 		out.T(out.Tip, "If you are still interested to make {{.driver_name}} driver work. The following suggestions might help you get passed this issue:", out.V{"driver_name": driver})
 		out.T(out.Empty, `
-	- Prune unused {{.driver_name}} images, volumes and abandoned containers.`, out.V{"driver_name": driver})
+	- Prune unused {{.driver_name}} images, volumes, networks and abandoned containers.
+		docker system prune --volumes`, out.V{"driver_name": driver})
```

Comment:
this only prunes volumes, how about prune everythign ?
and do you mind posting the Output after this PR ?
to make sure the indentation looks good ?

Reply:
> this only prunes volumes, how about prune everythign ?

This does prune everything, right?
```
$ docker system prune 
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all dangling images
  - all dangling build cache

Are you sure you want to continue? [y/N] n

$ docker system prune --volumes
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all volumes not used by at least one container
  - all dangling images
  - all dangling build cache

Are you sure you want to continue? [y/N] 
```
If you think the command is confusing, I agree, surely it should be something like: `docker system prune --include-volumes` ? :smile: 

> and do you mind posting the Output after this PR ?
to make sure the indentation looks good ?

done

> and is this command good on windows too ?

Pretty sure, but not certain. I'll update you if/when I confirm.

> also consider adding one line before and after the command to make it stand out for better copy pasting.

Done, should I add the same to https://github.com/kubernetes/minikube/blob/42792d34c22aee48611f31ee7c605a61f7efffcd/pkg/minikube/machine/advice.go#L60-L61 , for consistency and better copy and paste there too?

Reply:
> also consider adding one line before and after the command to make it stand out for better copy pasting.

I lost these changes during rebase off master. Need to retest that output is ok.